Examples/FireBaseUI/FireBaseUIViewController.swift

```diff
@@ -124,7 +152,11 @@ internal final class FireBaseUIViewController: ViewController {
 
     Task {
       do {
-        _ = try await Auth.auth().signIn(withEmail: email, password: password)
+        let result = try await Auth.auth().signIn(withEmail: email, password: password)
+        if let user = result.user {
+          let document = Firestore.firestore().document("users/\(user.uid)")
+          print("Made a document: \(document.path)")
```

Comment:
I wonder if we can surface this in the UI some how?

Reply:
Probably. I tried to use an `AlertController` but I saw that it's functionality is unimplemented, Is there a way for me to just call into some kind of Win32 alert?

Reply:
Yeah, I really need to get `AlertController` working ... in the mean time, you can just use the Win32 APIs using the `MessageBoxW` function through the `WinSDK` module. (There is an example in the Swift/Win32 UICatalog).

Reply:
Sadly it seems that the ability to call `MessageBoxW` is broken due to the WCHAR conversion issue with C++ interop we've seen elsewhere.

<img width="1123" alt="CleanShot 2023-09-21 at 11 57 04@2x" src="https://github.com/compnerd/swift-firebase/assets/63919/e91fe197-2df4-4d03-9205-4ce8cc778b53">